### PR TITLE
WRO-6459: Updated Silicon styles for Picker and RangePicker

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
- Updated silicon styles for Picker/RangePicker
- Fixed ss tests for Date and Time Picker - they were falsey passing because the value prop was different at each run, so there was no reference to compare to
- You may notice that some spacing(padding/margin) for Picker item from the design is not there and the height is different from the design.The padding was added to the container width/height to avoid ViewManager transition effect issues. Since those containers have transparent bg-color, the visual result is the same and the viewManager transition effect is not disturbed
- The layout with 5 items was kept for silicon skin. The UX guide is confusing because it shows picker with 3 and 5 values on the same screen. If we should switch to 3 items, let us know.
- `backgroundFocusClassName` was added in computed block in order for it to inherit the class name from `DateComponentPicker` component
- The z-indexes were set in order to obtain the following layout `active background --> focus background --> text`. There was an option to just set z-index:-1 for the .active background, but it would make the ss tests to fail because the active bakground would be covered by the ss tests view background. Any other working option would be welcomed

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-6459

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

File: styles/variables-silicon.less [hunk 1]
```diff
@@ -345,9 +345,14 @@
 
 // Picker
 // ---------------------------------------
-@agate-picker-font-size: 27px;
+@agate-picker-font-size: 33px;
 @agate-picker-font-weight: normal;
```
Comment (stanca-pop-lgp) [inline styles/variables-silicon.less:349]: It seems that in the design the font-weight is bold.

File: styles/variables-silicon.less [hunk 2]
```diff
@@ -345,9 +345,14 @@
...
 @agate-picker-active-font-weight: bold;
+@agate-picker-active-height: 99px;
+@agate-picker-horizontal-active-width: 201px;
+@agate-picker-horizontal-item-height: 160px;
+@agate-picker-horizontal-item-width: 249px;
+@agate-picker-horizontal-padding: 9px 6px;
 @agate-picker-item-height: 75px;
```
Comment (stanca-pop-lgp) [inline styles/variables-silicon.less:356]: Shouldn't this be 60px?

Comment (daniel-stoian-lgp) [reply styles/variables-silicon.less:356]: check PR description

File: styles/variables-silicon.less (hunk 1 above)
Comment (daniel-stoian-lgp) [reply styles/variables-silicon.less]: fixed

**Comment by stanca-pop-lgp** [comment]
Please add style for active value in disabled state.

File: internal/Picker/Picker.module.less [hunk 3]
```diff
@@ -73,33 +90,83 @@
...
+				z-index: 2;
+			}
+
+			&::before {
+				content: "";
+				background-color: @agate-picker-active-bg-color;
+				background-image: @agate-picker-active-bg-image;
+				position: absolute;
+				height: @agate-picker-active-height;
+				width: calc(100% - 2* @agate-picker-padding);
```
Comment (stanca-pop-lgp) [inline internal/Picker/Picker.module.less:121]: Please leave space after 2 😃

File: styles/variables-silicon.less [hunk 4]
```diff
@@ -345,9 +345,14 @@
...
 // Picker
 // ---------------------------------------
-@agate-picker-font-size: 27px;
-@agate-picker-font-weight: normal;
+@agate-picker-font-size: 33px;
+@agate-picker-font-weight: bold;
 @agate-picker-active-font-weight: bold;
+@agate-picker-active-height: 99px;
+@agate-picker-horizontal-active-width: 201px;
+@agate-picker-horizontal-item-height: 160px;
```
Comment (stanca-pop-lgp) [inline styles/variables-silicon.less:353]: Please add multiple of 3 -> 159px

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by seunghoh — CHANGES_REQUESTED** [review]
Let's revisit later when we are requested.
I just check request changes not for mering accidently

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM 👍

**Review by seunghoh — APPROVED** [review]

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Pull request merged**